docs/source/flyingpigeon_tutorial.rst

```diff
@@ -0,0 +1,75 @@
+
+.. code:: ipython3
```

Comment:
We can use notebooks to show examples.

Maybe create a new repo on github for notebooks and just reference in the docs?